rpi5.yaml

```diff
@@ -342,3 +345,24 @@ parameters:
                 type: raw_image
                 size: 8 GiB
                 image_path: "%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{MACHINE}/%{XT_DOMD_IMAGE}-%{MACHINE}.rootfs.ext4"
+
+    nvme:
+      overrides:
+        components:
+          domd:
+            builder:
+              conf:
+                - [DOMD_OVERLAYS, "%{SOC_FAMILY}-%{MACHINE}-pcie1.dtbo"]
+                - [MACHINE_FEATURES:append, " domd_nvme"]
+                - [DOMD_BOOTARGS, "console=ttyAMA0 earlycon=xen earlyprintk=xen clk_ignore_unused root=\\\\/dev\\\\/nvme0n1p1 rootfstype=ext4 rootwait"]
```

Comment:
I believe this is a good time to factor out `DOMD_BOOTARGS` from this option. Just add new variable `DOMD_ROOTFS_DEV` and set 

```
[DOMD_BOOTARGS, "console=ttyAMA0 earlycon=xen earlyprintk=xen clk_ignore_unused root=%{DOMD_ROOTFS_DEV} rootfstype=ext4 rootwait"]
```

in common code. Both here and in `usb` case.

Reply:
Fixed